middleware.ts

```diff
@@ -9,9 +8,16 @@ import CredentialsProvider from "next-auth/providers/credentials";
 import NextAuth, { Session } from "next-auth";
 import { JWT } from "next-auth/jwt";
 
-acceptLanguage.languages(languages);
 const localPathRegEx = new RegExp("^(?!((?:[a-z+]+:)?//))", "i");
 
+const verboseDebug = false;
+
+const debugLogger = async (message: string) => {
+  if (verboseDebug) {
```

Comment:
Is this for local debugging to flip it to true or something?

Reply:
Yes, I found the middleware debug logs to be really noisy when not trying to debug what's going on with the routing but also really helpful if ever someone is having issues.  I thought about using a env var flag to trigger more verbose logging but was worried it might just be forgotten about.  So I went with a local variable in the file.  This way you can also switch it on the fly without having to restart your dev environment.